maintenance-mode.php

```diff
@@ -30,3 +32,15 @@
 		exit;
 	} );
 }
+
+/**
+ * Localize plugin
+ *
+ * @since 0.1.1
+ * @uses load_plugin_textdomain
+ * @return void
+ */
+function vip_maintenance_mode_load_plugin_textdomain() {
+	load_plugin_textdomain( 'maintenance-mode', FALSE, basename( dirname( __FILE__ ) ) . '/languages/' );
+}
+add_action( 'plugins_loaded', 'vip_maintenance_mode_load_plugin_textdomain' );
```

Comment:
Thanks, this is now updated. I followed the example in the [Plugin Handbook](https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/). In the Codex and in the Code Reference, the `init`hook is used.